src/App.tsx

```diff
@@ -1,223 +1,81 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Errors } from './components/Errors';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Loader } from './components/Loader';
+import { TodoList } from './components/TodoList';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
```

Comment:
Let's move this useRef with the corresponding `useEffect` to the `Header` component. Looks like you need this logic only in `Header` component, not in the `App.tsx`